### ISSUE TITLE
Dashboard not opening in emacsclient frames

### ISSUE THREAD

**Issue by vsr625** [hlissner/.emacs.d#111]
When using emacs in daemon mode, the client emacs window starts up with a scratch buffer rather than the doom dashboard. I tried this with a fresh emacs config from the main branch. Can you please look into this? 

And one more question, what is it about switching from linum to nlinum, then back to linum, and recently again moving back to nlinum mode for line numbers. 

And the newer modeline looks kinda neat. Keep up with the good work.

**Comment by hlissner** [comment]
> When using emacs in daemon monde, the client emacs window starts up with a scratch buffer rather than the doom dashboard

I've pushed a likely fix in https://github.com/hlissner/.emacs.d/commit/a3adc1de2922fb20de677eae66cfc960e1270fd9. Give it a try and let me know if you encounter any issues.

> what is it about switching from linum to nlinum, then back to linum, and recently again moving back to nlinum mode for line numbers.

The initial switch was for speed. The recent back-and-forth was because I jumped the gun.

`linum-mode` is excruciatingly slow under common circumstances (and this is on a beefy i7+16gb+SSD rig). When #59 opened, I wondered if linum was as bad as I remembered it. A lazy test-run with linum produced no issues, so I assumed its issues were somehow fixed between Emacs 24.4 and 25.

But that was short lived. gilbertw1 and I found workarounds for nlinum's issues and I rediscovered why I had left linum in the first place. And back to nlinum we went!
